apps/src/codebridge/Console/Console.tsx

```diff
@@ -5,13 +5,19 @@ import {FitAddon} from '@xterm/addon-fit';
 import {ImageAddon} from '@xterm/addon-image';
 import {Terminal} from '@xterm/xterm';
 import React, {useCallback, useEffect, useRef, useState} from 'react';
+import {useDispatch} from 'react-redux';
 
 import codebridgeI18n from '@cdo/apps/codebridge/locale';
+import {FontSize} from '@cdo/apps/lab2/constants';
```

Comment:
Confirmed with @samantha-code that we want same font sizes as we have for code editor - [Slack discussion](https://codedotorg.slack.com/archives/C06JELCAPT9/p1744059743542899).